lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,18 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
```

Comment:
Added https://github.com/distribusion/supa/pull/21/commits/5eda8d3e83f7f9f3a193ef2c2a53489acc0de128. I had to change the interface of this command to make it work with options. Now it is

```ruby
attributes [:attribute_1, :attibute_2], { hide_when_nil: true }
```